### PR TITLE
MTA-145: changing the references to earlier versions

### PR DESCRIPTION
**Pull request by anarnold97**
[MTV-145](https://issues.redhat.com/browse/MTA-145)

Changing the document-attributes.adoc file for moving from 6.0.1 release to 6.1.0 release

### CODE REVIEW

File: docs/topics/templates/document-attributes.adoc [hunk 1]
```diff
@@ -13,19 +13,19 @@
...
 :WebNameTitle: User Interface
 :WebNameURL: user_interface_guide
 :WebConsoleBookName: {WebNameTitle} Guide
-:ProductVersion: 6.0.1
+:ProductVersion: 6.1.0
 :PluginName: MTA plugin
-:MavenProductVersion: 6.0.1.GA-redhat-00006
-:ProductDistributionVersion: 6.0.1.GA-redhat
-:ProductDistribution: mta-6.0.1.GA-offline.zip
+:MavenProductVersion: 6.1.0.GA-redhat
```
Comment (anarnold97) [inline docs/topics/templates/document-attributes.adoc:26]: @sshveta - the link you sent did not look it gave the right Maven reference. 
Please can I double check as I need to be certain we have the correct references?

File: docs/topics/templates/document-attributes.adoc [hunk 2]
```diff
@@ -13,19 +13,19 @@
...
+:ProductDistributionVersion: 6.1.0.GA-redhat
+:ProductDistribution: mta-6.1.0.GA-offline.zip
```
Comment (anarnold97) [inline docs/topics/templates/document-attributes.adoc:28]: @sshveta - again the you sent did not look like it was correct.

Please can I ask you to double-check before I do anything further?

Thanks

File: docs/topics/templates/document-attributes.adoc (hunk 1 above)
Comment (anarnold97) [reply docs/topics/templates/document-attributes.adoc:26]: @dymurray - please can i ask for your help on this and please can you confirm the following values:

:MavenProductVersion: 6.0.1.GA-redhat-00006
:ProductDistributionVersion: 6.0.1.GA-redhat
:ProductDistribution: mta-6.0.1.GA-offline.zip

Ta muchly

**Review by sshveta — APPROVED** [review]
Looks correct to me .
Tagging other to check

File: docs/topics/templates/document-attributes.adoc (hunk 1 above)
Comment (dymurray) [reply docs/topics/templates/document-attributes.adoc:26]: Yes, looks good to me.

Comment (dymurray) [reply docs/topics/templates/document-attributes.adoc:26]: To be clear, when I said "looks good to me", those are the 6.0.1 values. 

For 6.1.0 the changes in the PR looks sane. @sshveta if you can confirm the actual maven product reference that'd be great.

Comment (dymurray) [reply docs/topics/templates/document-attributes.adoc:26]: @mrizzi may also know the answer to this.

Comment (sshveta) [reply docs/topics/templates/document-attributes.adoc:26]: It's MTA-6.1.0.GA-maven
Screenshot from 2023-04-21 08-43-56

Comment (mrizzi) [reply docs/topics/templates/document-attributes.adoc:26]: @anarnold97 the value is partial, the complete one is
```suggestion
:MavenProductVersion: 6.1.0.GA-redhat-00005
```

**Review by mrizzi — DISMISSED** [review]
Provided the full version for the Maven Plugin.

**Comment by sbeskin-redhat** [comment]
@anarnold97 Looks fine (provided the info is correct)

**Pull request merged**